src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -1173,12 +1173,6 @@ public void setWrapGoogModulesForWhitespaceOnly(boolean enable) {
   List<String> packageJsonEntryNames;
 
   /**
-   * Needed by {@link RewriteJsonToModule}, but defined here because RewriteJsonToModule is not
-   * part of the core build.
-   */
-  public static final String PACKAGE_JSON_MAIN = "main";
```

Comment:
this is not actually used anywhere else, since we generalized package.json fields.